ui/src/components/ResourceDialog/ProcessorsDialog/ProcessorForm.tsx

```diff
@@ -21,7 +22,15 @@ export const ProcessorForm: React.FC<Props> = ({ parameterDefinitions }) => {
             Configure
           </Typography>
         </Grid>
-
+        <Grid item xs={6}>
+          <ResourceDisplayNameInput
+            value={formValues.display_name}
+            onValueChange={(v: string) =>
+              // setFormValues((prev) => ({ ...prev, name: v }))
+              {}
```

Comment:
We're going to want this. We're unable to set the display name when creating now.
```suggestion
              setFormValues((prev) => ({ ...prev, name: v }))
              
```